### DISCUSSION TITLE
Not getting application details in signoz frontendui or on signozUi after configuring the opentelemetry with give java sample app

### DISCUSSION THREAD

**Discussion by Chandu5662** [SigNoz/signoz#5269] [Q&A]
Hi All,Good Day!!
I have installed signoz with docker compose then all the container are started next i was trying to instrumenting java app with opentelemetry to my java application , i did that same process which they have provided i did the same process but In UI the new sample app service was not even showing or nothing it showing

**Comment by nitishfy** [comment]
Hi, can you answer couple of questions that i've listed below:

1. Which tutorial are you referring to?  Is this the [one](https://signoz.io/docs/instrumentation/java/)?
2. Can you share your `otel-config.yml` file?
3. Did you check if all the containers are running well? Did you check the logs of the container - especially the clickhouse container?
